RELEASE-NOTES.md

```diff
@@ -1,5 +1,79 @@
 # Prefect Release Notes
 
+## Release 2.13.2
+
+### Server-side enforcement of deployment parameter schemas
```

Comment:
```suggestion
### Opt-in to server-side enforcement of deployment parameter schemas
```
or alternatively, "This opt-in feature"

but I think it could be good to mention that